src/Zynga/Framework/Performance/V1/XHProfiler.hh

```diff
@@ -5,14 +5,20 @@ namespace Zynga\Framework\Performance\V1;
 use Zynga\Framework\Environment\CodePath\V1\CodePath;
 
 require_once
-  CodePath::getRoot().'/third-party/xhprof/xhprof_lib/utils/xhprof_lib.php'
+  CodePath::getExternalLibraryPath().
```

Comment:
minor naming thing: maybe getExternalLibraryRoot is more consistent with sibling getRoot?

Reply:
Sure. Will change!

Reply:
Done!